src/http/one/RequestParser.cc

```diff
@@ -14,11 +14,17 @@
 #include "SquidConfig.h"
 
 Http1::Parser::size_type
-Http::One::RequestParser::firstLineSize() const
+Http::One::RequestParser::RequestLineSize(const size_t methodLength, const size_t requestTargetLength)
```

Comment:
See Http::One::ResponseParser::StatusLineSize() comment regarding the name change here.

Reply:
> Please drop these changes.

Done in branch commit dd6fc48.